modules/lx-instance/watchmaker-lx-instance.template.cfn.json

```diff
@@ -1544,10 +1544,10 @@
                                 "yum -y install gcc python-devel libffi-devel openssl-devel\n",
                                 "pip install",
                                 " --index-url=\"$PYPI_URL\"",
-                                " --upgrade cffi\n",
+                                " --upgrade 'pycparser<2.19;python_version<\"2.7\"' cffi\n",
                                 "pip install",
                                 " --index-url=\"$PYPI_URL\"",
-                                " --upgrade 'cryptography<2.2;python_version<\"2.7\"' 'cryptography;python_version>=\"2.7\"'",
+                                " --upgrade 'pycparser<2.19;python_version<\"2.7\"' 'cryptography<2.2;python_version<\"2.7\"' 'cryptography;python_version>=\"2.7\"'",
```

Comment:
Is `pycparser<2.19;python_version<\"2.7\"` necessary in the second `pip install`?